mahjong/src/utils/index.js

```diff
@@ -1,32 +1,24 @@
-export const generateUniqueArray = (arrayLength, maxNumber) => {
-
-    let randomNumbers = [];
-
-    function generateNumber(maxNr) {
-        let random = (Math.random() * maxNr).toFixed();
-
-        random = Number(random);
-
-        if(!randomNumbers.includes(random)) {
-            randomNumbers.push(random);
-            return random;
-        } else {
-            if(randomNumbers.length < maxNr) {
-            return  generateNumber(maxNr);
-            } else {
-            console.log('No more numbers available.')
-            return false;
-            }
-        }
+const isPrime = num => {
+    if (num <= 1) return false;
+    if (num % 2 === 0 && num > 2) return false;
+    const s = Math.sqrt(num);
+    for (let i = 3; i <= s; i += 2) {
+      if (num % i === 0) return false;
     }
-
-    const newArray = Array.from(Array(arrayLength), () => generateNumber(maxNumber));
-
-    return newArray;
-}
-
-export const shuffle = (array) => {
+    return true;
+};
+  
+  const shuffle = array => {
     const newArray = [...array];
     newArray.sort(() => Math.random() - 0.5);
     return newArray;
-}
\ No newline at end of file
+};
+  
+  export const generateShuffledPrimeNumbers = () => {
+    const primeNumbers = Array.from(Array(60), (_, i) => i + 1).filter(num =>
+      isPrime(num),
+    );
+    const shuffledSliced = shuffle(primeNumbers).slice(0, -1);
+    return shuffle(shuffledSliced.concat(shuffledSliced));
```

Comment:
Why do you shuffle twice? 

You can also use `[...array, ...array2]` to merge two arrays